app/api/csv/specs/csvLoaderLanguages.spec.ts

```diff
@@ -13,11 +12,7 @@ import fixtures, { template1Id } from './fixtures';
 import { createTestingZip } from './helpers';
 
 const removeTestingZip = async () =>
-  new Promise(resolve => {
-    fs.unlink(path.join(__dirname, 'zipData/testLanguages.zip'), () => {
-      resolve();
-    });
-  });
+  filesystem.deleteFile(path.join(__dirname, 'zipData/testLanguages.zip'));
```

Comment:
@mfacar its up to the developer, filesystem is our own custom functions, while fs is the nodejs raw api, (the promisified one).

in this particular case unlink will throw an error if the file does not exists while deleteFile will ignore that particular error.

im not really happy with this because its not really clear what to use, but i can not think of a better way, any suggestions ? anyways its an improvement over the previous version.

Reply:
It makes sense if we don't want to have the catch everywhere, definitively it's a great improvement.  Thanks for the clarification 👍